packages/x-charts/src/ChartsSurface.tsx

```diff
@@ -18,6 +21,7 @@ export interface ChartsSurfaceProps {
   sx?: SxProps<Theme>;
   children?: React.ReactNode;
   disableAxisListener?: boolean;
+  onClick?: OnLineClick;
```

Comment:
I assume you plan to go with 

```ts
onClick?: OnLineClick | OnBarClick | OnScatterClick;
```

I'm wondering if we coudl not do 

```ts
onClick?: (event: React.MouseEvent<SVGElement>, params?: LineClickParams | BarClickParams | ScatterClickParams)
```

For PieChart it was not an issue since the `onClick` was passed to `<PiePlot/>` which is not a DOM element. It's a React.Fragment with multiple custom element. But here it's `<Surface/>` which correspond to the `<svg />` which could get a native `onClick`